pdf/pdfrenderthread.cpp

```diff
@@ -86,18 +87,25 @@ class SearchThread: public QThread
             }
 
             delete page;
+
+            if ((i + 1) % 3 == 0 || i + 1 == m_document->numPages()) {
+                emit searchProgress(float(i) / float(m_document->numPages() - 1),
```

Comment:
Ah, my bad. The if initially didn't have the ```|| i + 1== numPages()``` part, so the division by zero case cannot happen. Later, when I added the or case in the if, I didn't remember the division by zero case…

Thanks for spoting, Corrected.